lib/publiccloud/instance.pm

```diff
@@ -848,5 +848,25 @@ sub do_systemd_analyze_time {
     return @ret;
 }
 
+sub retry_on_script_output {
```

Comment:
I'm afraid this is a very generic name for a function that runs only a hard-coded command. =/

Reply:
Sorry, but this coommand was not intended here to have a retry: the failure is managed into the while loop!
Why do you need this retry now?

Reply:
I cannot read the SD ticket: why do you think this change is needed?